.github/workflows/release-master.yml

```diff
@@ -0,0 +1,55 @@
+# if changing this name, also update promotion.yaml
+name: release-master
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  golint:
+    name: vet fmt and lint
+    runs-on: ubuntu-20.04
+    container: golang:1.13.7
+    steps:
+    - uses: actions/checkout@v2.3.3
+    - name: Install golint
+      run: go get -u golang.org/x/lint/golint
+    - run: make lint
+      shell: bash
+  gotest:
+    name: go test
+    runs-on: ubuntu-20.04
+    container: golang:1.13.7
+    steps:
+    - uses: actions/checkout@v2.3.3
+    - run: make test
+  docker_build:
+    name: docker_build
+    runs-on: ubuntu-20.04
+    container:
+      image: docker:20.10.5
+      options: -t
+    # Setting up dind service container
+    services:
+      docker:
+        image: docker:20.10.5-dind
+        env:
+          DOCKER_DRIVER: overlay
+          DOCKER_HOST: tcp://localhost:2375
+    steps:
+    - name: Install Tools
+      run: apk add --update make git jq rsync curl
```

Comment:
I think that some of these are present already?

Reply:
```
$ docker run -ti docker:20.10.5 sh                                                   
/ # make
sh: make: not found
/ # git
sh: git: not found
/ # jq
sh: jq: not found
/ # rsync
sh: rsync: not found
/ # curl
sh: curl: not found
```

Apparently they are not.

Reply:
Ahh, my bad. They are in the github actions ubuntu image. Not the docker image...